src/components/ageAssurance/useAgeAssuranceCopy.ts

```diff
@@ -10,6 +10,9 @@ export function useAgeAssuranceCopy() {
       notice: _(
         msg`The laws in your location require that you verify your age before accessing certain features on Bluesky like adult content and direct messaging.`,
       ),
+      banner: _(
+        msg`The laws in your location require you verify your age. Tap to learn more.`,
```

Comment:
```suggestion
        msg`The laws in your location require that you verify your age. Tap to learn more.`,
```
nit: I think it would read slightly better with `that`, like the string above.